src/pkg/cli/compose/validation.go

```diff
@@ -280,8 +279,7 @@ func ValidateProject(project *compose.Project) error {
 }
 
 func warnf(format string, args ...interface{}) {
-	logrus.Warnf(format, args...)
-	term.SetHadWarnings(true)
+	term.Warnf(format, args...)
```

Comment:
should we still bother with this `warnf` function? Confusing.

Reply:
Fixed